olp-cpp-sdk-core/src/network2/NetworkSettings.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include <olp/core/network2/NetworkSettings.h>
```

Comment:
Use "" for local includes

Reply:
That is not a local include, isn't it? and looks much nicer that 
```
#include "../../olp/core/network2/NetworkSettings.h"
```

Reply:
You don't need to use a relative path as this is managed by cmake during configuration:

```suggestion
#include "olp/core/network2/NetworkSettings.h"
```

Reply:
cmake manages paths that are visible through ```#include<>```.  ```#include""``` is intended to look into current directory and just as a fallback-mode and in case nothing was found it also works as ```#include<>```. 
In this particular case there is nothing to fear, but making quoted include a convention smells. Consider this:
```
#include "utils/json.h"
```
* How many directories called utils are there? 
* What if somebody places their utils into ```pwd()``` ?  

I'd prefer to leave it this way.

Reply:
Then we need to change all other files also, as every class now includes its own header via "".
And yes, we have an utils folder but under `olp/core/utils/` and it will stay there i presume.
So either one way or the other.

Reply:
consistency. ok.